python/bin/repl_stub.py

```diff
@@ -0,0 +1,3 @@
+import code
+
+code.interact()
```

Comment:
What is `code` and what is `code.interact`? Is it something special? Could you please add a comment?